diesel_cli/src/errors.rs

```diff
@@ -58,4 +58,8 @@ pub enum Error {
     ColumnLiteralParseError(syn::Error),
     #[error("Failed to parse database url: {0}")]
     UrlParsingError(#[from] url::ParseError),
+    #[error("Failed to parse CLI parameter: {0}")]
+    ClapMatchesError(#[from] clap::parser::MatchesError),
+    #[error("No print schema key with the name `{0}` exists")]
```

Comment:
Can we point out more explicitly that this refers to `[print_schema.{0}]` entries in your `diesel.toml` file?